src/magellan_core/launch/main.launch

```diff
@@ -0,0 +1,5 @@
+<launch>
```

Comment:
Is this intended to be the main launch file for the entire robot? I created one in my WIP deployment PR, so I'll update it if that's the intent.

Reply:
I was imagining it being the file that launches everything that needs launched every time and have higher launch files that launch specific scripts in tandem with main.launch